docs-source/spring/content/development/cli/_index.md

```diff
@@ -66,7 +66,7 @@ An application is a namespace encompassing related Microservices. For example, a
 
 The `create` command results in the creation of an application namespace (Kubernetes *namespace*). The application namespace provides a mechanism for isolating groups of resources, especially the Microservices.
 
-The `delete` command results in the delete of an application namespace (Kubernetes *namespace*) completely or a specific Microservice. <ins>**Be carefully**</ins> because it is not possible rollback the components deleted.
+The `delete` command results in the delete of an application namespace (Kubernetes *namespace*) completely or a specific Microservice. **Be careful** because it is not possible rollback the components deleted.
```

Comment:
The delete command results in the complete deletion of an application namespace (Kubermetes namespace) or for a specific Microservice. Ensure that you want to delete the application namespace. You cannot rollback the components once deleted.

Reply:
According to Mark's comment, "Microservice" should be "microservice".